docs/features/containers.md

```diff
@@ -357,6 +357,16 @@ const container = await new GenericContainer("alpine").start();
 await container.restart();
 ```
 
+## Committing a container to an image
+
+```javascript
+const container = await new GenericContainer("alpine").start();
+// Do something with the container
+await container.exec(["sh", "-c", `echo 'hello world' > /hello-world.txt`]);
+// Commit the container to an image
+const newImage = await container.commit({ repository: "my-repo", tag: "my-tag" });
```

Comment:
What is `newImage` here? `container.commit` returns `Promise<void>`

Reply:
Thanks, good catch.  It was supposed to return the image ID (updated)